server/utils/nunjucksSetup.ts

```diff
@@ -389,6 +390,21 @@ export function registerNunjucks(app?: express.Express): Environment {
     return legalStatus[status]
   })
 
+  njkEnv.addFilter(
+    'shouldShowHardStopWarning',
+    (licence: { kind: LicenceKind; hardStopWarningDate: string; hardStopDate: string }): boolean => {
+      const now = startOfDay(new Date())
+      return (
+        config.hardStopEnabled &&
+        licence.kind !== LicenceKind.VARIATION &&
+        !!licence.hardStopWarningDate &&
+        parseCvlDate(licence.hardStopWarningDate) <= now &&
+        !!licence.hardStopDate &&
+        now < parseCvlDate(licence.hardStopDate)
```

Comment:
I think I'd probably prefer to move the checks of `hardStopWarningDate` and `hardStopDate` definition to outside of the big return statement, just for tidiness and to reduce the size of that single check
```
if (!licence.hardStopWarningDate || !licence.hardStopDate) { return false }
return ...
```
Happy to have a discussion about it if you disagree!